### PR TITLE
Crashdump Writer plugin

### PR DESCRIPTION
**Pull request by paulkermann**
This PR furthers the work done in #472, to correctly create a crashdump (properly loaded in Windbg).
Importantly, this also allows parsing encoded KDBG structs in `framework/plugins/windows/info.py`.

### CODE REVIEW

File: volatility3/framework/plugins/windows/info.py [hunk 1]
```diff
@@ -7,11 +7,28 @@
 
 from volatility3.framework import constants, interfaces, layers, symbols
 from volatility3.framework.configuration import requirements
-from volatility3.framework.interfaces import plugins
+from volatility3.framework.interfaces import plugins, configuration
 from volatility3.framework.renderers import TreeGrid
 from volatility3.framework.symbols import intermed
 from volatility3.framework.symbols.windows import extensions
+from volatility3.framework.layers import physical
+import struct
```
Comment (digitalisx) [inline volatility3/framework/plugins/windows/info.py]: I recommend that arrange the modules in alphabetical order.
And recommended that the local package, not the module (API) created by Volatility, be directed to the top.

For Example:
```
import struct

from volatility3.framework.interfaces import configuration, plugins # Point
from volatility3.framework.layers import physical # Point
from volatility3.framework.renderers import TreeGrid
from volatility3.framework.symbols import intermed
from volatility3.framework.symbols.windows import extensions
```

**Review by digitalisx — COMMENTED** [review]
I reviewed a very small thing about coding conventions/formats by PEP 8 style!

File: volatility3/framework/plugins/windows/info.py (hunk 1 above)
Comment (paulkermann) [reply volatility3/framework/plugins/windows/info.py]: I have moved the import

**Comment by paulkermann** [comment]
This PR is rebased on develop so only something like the last 3 commits really matter

File: volatility3/framework/plugins/windows/info.py (hunk 1 above)
Comment (digitalisx) [reply volatility3/framework/plugins/windows/info.py]: Thank you for reflecting my opinion!

**Comment by paulkermann** [comment]
#718 is now the active PR for this plugin

**Pull request closed without merging**